lbrynet/lbrynet_daemon/UIManager.py

```diff
@@ -61,9 +61,12 @@ def __init__(self, root):
                 self.loaded_requirements = None
 
     def setup(self, branch=None, check_requirements=None, user_specified=None):
-        local_ui_path = settings.local_ui_path or user_specified
-        self.branch = settings.ui_branch or branch
-        self.check_requirements = settings.check_ui_requirements or check_requirements
+        local_ui_path = (user_specified if user_specified is not None
```

Comment:
I prefer the succinctness of:

`local_ui_path = user_specified or settings.local_ui_path`

And its a common enough pattern for dealing with optional arguments that it is also more readable.